bubble-wrap-style.yaml

```diff
@@ -5655,8 +4946,54 @@ layers:
                kind: beach
            draw:
                dots:
-                   color: [0.780,0.740,0.538]
-                   visible: true
+                    color: [0.780,0.740,0.538]
+                    visible: true
+
+        plant_nursery:
+            filter:
+                kind: plant_nursery
+            draw:
+                dots:
+                    color: [0.955,0.752,0.650]
+                    visible: true
```

Comment:
Are all these `visible: true` lines turning on something that's turned off way up above?

Reply:
Yep